packages/flutter/lib/src/cupertino/sliding_segmented_control.dart

```diff
@@ -924,30 +983,73 @@ class _RenderSegmentedControl<T extends Object> extends RenderBox
   }
 
   Size _calculateChildSize(BoxConstraints constraints) {
+    final double childWidth = _getMaxChildWidth(constraints);
```

Comment:
is the method still needed?

Reply:
Removed `_calculateChildSize` and updated `computeDryBaseline` method:)